common/api.py

```diff
@@ -230,7 +233,7 @@ def _is_valid_test_name(cls, name):
         if name is None:
             return False
 
-        return name.lower().startswith('test_')
+        return name.lower().startswith('test_') or name.lower().endswith('_test')
```

Comment:
This could be refactored to use the _is_test_notebook_name method suggested above.